index.js

```diff
@@ -11,6 +11,9 @@ module.exports = {
           enabled: this._shouldIncludeFiles(),
           import: ['index.js']
         };
+      },
+      jquery: {
+        import: ['dist/jquery.js'],
```

Comment:
This needs to be verified. 
Anyways seems like it's not a big deal cause it's for testing only.
Also this way we can guarantee that we don't consume jquery from ember